ergo/distributions/conditions.py

```diff
@@ -123,6 +185,16 @@ def loss(self, dist):
             total_loss += scale_penalty
         return self.weight * total_loss
 
+    def get_normalized(self, true_min: float, true_max: float):
+        # I think this is wrong and that you do need to scale to normalize,
```

Comment:
@stuhlmueller is this comment accurate? I.e. do we need implement normalization for the 2 conditions you added?

(I think we should do it on a future PR if so, since they're not actually used in the code yet)

Reply:
Yep, it's definitely wrong, fixing now